test/conftest.py

```diff
@@ -11,11 +13,18 @@ def pytest_configure(config):
     config.addinivalue_line("markers", "emulation: mark test requiring qemu binfmt_misc to run")
 
 
-def pytest_collection_modifyitems(config, items):
+def pytest_collection_modifyitems(config, items) -> None:
     if config.getoption("--run-emulation"):
         # --run-emulation given in cli: do not skip emulation tests
         return
     skip_emulation = pytest.mark.skip(reason="need --run-emulation option to run")
     for item in items:
         if "emulation" in item.keywords:
             item.add_marker(skip_emulation)
+
+
+@pytest.fixture(
+    params=[{"CIBW_BUILD_FRONTEND": "pip"}, {"CIBW_BUILD_FRONTEND": "build"}], ids=["pip", "build"]
+)
+def build_mode(request) -> Dict[str, str]:
```

Comment:
Maybe this fixture would be better named as `build_frontend` or `build_frontend_option`?

Reply:
I used `build_frontend_env`, since it's an env dict.